addons/sales_team/views/crm_team_views.xml

```diff
@@ -53,7 +53,7 @@
                     <notebook>
                         <page string="Members" name="members_users">
                             <field name="member_ids" mode="kanban"
-                                class="w-100" context="{'allow_create_employee': False}">
+                                class="w-100">
```

Comment:
This one seems to work ok, but again it changes the behavior introduced recently in b47f73bf4ec14d5d95ea567fc62ce05c58ea58a3
-> so it's not legacy behavior and should ideally not be changed.